### PR TITLE
Login component tj

### PR DESCRIPTION
**Pull request by taylorjohnson141**
## Types of changes 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:  
- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

What’s this PR do?

This PR adds the login component to the application

Where should the reviewer start?
Login.js

How should this be manually tested?
Should not be tested yet

Any background context you want to provide?
We will add the Form validation later

What are the relevant tickets?
N/A as need to connect to header to know if it works
Screenshots (if appropriate)
Questions:

### CODE REVIEW

File: src/Login.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+const { Component } = require("react")
```
Comment (chadrick-d-dev) [inline src/Login.js:2]: Would like to discuss line 2, and what it is doing.

File: src/Login.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  constructor(){
+    super()
+    this.state = {
+      email:'',
+      name:'',
+      error :''
+    }
+  }
+  
+  handleChange(){
```
Comment (chadrick-d-dev) [inline src/Login.js:14]: let's review this

File: src/Login.js (hunk 1 above)
Comment (taylorjohnson141) [reply src/Login.js:2]: It's another way to require the component class in react!

File: src/Login.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    let name = event.target.name
+    this.setState({[name]:event.target.value})
+  }
+  render() {
+    return (
+      <form onSubmit = {this.handleSubmit}>
```
Comment (chadrick-d-dev) [inline src/Login.js:20]: let's review this piece this.handleSubmit

**Pull request merged**